### ISSUE TITLE
Some settings do not restore on startup

### ISSUE THREAD

**Issue by jgrubs1** [wkpark/uddetailer#78]
Certain DD settings do not restore after restarting A1111. These include:

1. Settings for Secondary model. Particularly, this setting: Inpaint B detections > before inpainting A. As a result, even if a secondary model is configured, and default UI settings are saved and appear in the ui-config.json file (as     "customscript/ddetailer.py/txt2img/Inpaint B detections/visible": true, "customscript/ddetailer.py/txt2img/Inpaint B detections/value": "before",), in reality the setting does not show up in the UI after restarting A111, and the second model correctly *detects* but does not *inpaint* anything (because Inpaint B detections setting is not restored).

2. Object classification settings are also not restored and do not appear after relaunching A111. I have mentioned that in a comment to this issue: https://github.com/wkpark/uddetailer/issues/76 , regardless of whether they are set at a primary or secondary model. 
 
In other words, uddetailer correctly saves its settings, but does not restore some of them. In order to have these settings re-appear (and to start working), one has to select a different model from the list, then re-select the original model. Only then the correct settings will show up.

Here's what I see after restarting A1111:
<img width="889" alt="no-A-B" src="https://github.com/wkpark/uddetailer/assets/80319126/ec5ff80a-d330-406e-aeb2-267fc2659cb5">

To have object class "hand" back, and to make the secondary model work, I have to re-select it from the list (switch to another model, switch back to yolo/hand):
<img width="895" alt="must-reload" src="https://github.com/wkpark/uddetailer/assets/80319126/7031b2e7-386d-428f-ae64-9a56e73bb553">

**Comment by wkpark** [comment]
thank you for your reporting!

I will look into this issue asap.
